controllers/apply/standard-proposal/fields.js

```diff
@@ -149,7 +149,7 @@ module.exports = function fieldsFor({ locale, data = {}, flags = {} }) {
              * pending proposals that used checkbox selection.
              *
              * Multiple selection disabled until UK-portfolio is enabled.
-             * @TODO: Remove and switch back to checkbox when launching UK-portfolio
```

Comment:
Removed the TODO as this is more a longer-term product decision when we launch the UK-portfolio. Nothing _actionable_ in this code.